docs/overview/streaming.md

```diff
@@ -0,0 +1,51 @@
+---
+id: overview_streaming
+title: "Streaming"
+---
+
+Zio-elastic search is streaming friendly library and there are few specific APIs that are used for creating ZIO streams. 
+Library makes use of two Elasticsearch APIs to create ZIO Stream, and those are Scroll API and Search_after API. 
```

Comment:
Suggestion:
"The library offers two different streaming modes relying on two different ways of retrieving paged results from Elasticsearch: `scroll` and `search_after`."